// const promise = new Promise((resolve, reject) => {
//   setTimeout(() => {
//       console.log("Got user");

//      // resolve({ user: "Moti" });
//       reject(new Error("Some generic error"))
//   }, 2000);
// });

// promise.then(user => {
//   console.log(user);
// }).catch(err => console.log(err.message));
